.github/workflows/pytest.yml

```diff
@@ -49,21 +49,24 @@ jobs:
           echo SQLALCHEMY_POOL_PRE_PING=True >> ./vars.env
           echo SQLALCHEMY_POOL_RECYCLE=3600 >> ./vars.env
           cat alws/alembic.ini | sed 's/almalinux-bs/test-almalinux-bs/' | tee alws/alembic.ini
-      - uses: isbang/compose-action@v1.5.1
-        name: Run docker-compose
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build Docker image
+        uses: docker/build-push-action@v5
         with:
-          # Use this if you change something in Dockerfile
-          # up-flags: --build --no-deps
-          up-flags: "--pull always --no-build --no-deps"
-          down-flags: "--volumes"
-          services: |
-            db
-            web_server
+          context: .
+          load: true
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+      - name: Start services
+        run: docker compose up -d --no-deps web_server db
       - name: Run unit tests (pytest)
-        run: docker-compose run -v /tmp:/tmp --no-deps web_server bash -o pipefail -c '
-          pytest -vv --ignore alws/ --cov-report term-missing:skip-covered
```

Comment:
why we no longer have to ignore the `alws` folder?

Reply:
Oh, those `pytest` parameters should go into `pyproject.toml` later. Rolled the command line back to what it was.